betamax/serializers/proxy.py

```diff
@@ -27,6 +27,13 @@ def __init__(self, serializer, cassette_path, allow_serialization=False):
         self.cassette_path = cassette_path
 
     def _ensure_path_exists(self):
+        directory, _ = os.path.split(self.cassette_path)
+        if directory != '':
+            if not os.path.isdir(directory):
+                raise OSError(
```

Comment:
I'm fine with RuntimeError.

I went with just giving a clear error message, since it's easy enough to mkdir if you know what the problem is. Doing it this way gives most of the benefit without making it too magic. I'm not familiar enough with the normal usage patterns of betamax, but creating a new cassette directory is not particularly frequent for a particular project? If so, I think keeping it as just an error is okay.